src/Tools/LinkabilityChecker/LinkabilityChecker.csproj

```diff
@@ -13,6 +13,7 @@
 
   <ItemGroup>
     <!-- Packages required to produce a complete dependency graph for the trimmer -->
+    <Reference Include="System.Configuration.ConfigurationManager" />
```

Comment:
Why does unchanged code need a new dependency e.g., is this related to the src/Shared/TrimmingAttributes.cs changes❔

Reply:
This project references everything in [TrimmableProjects.props](https://github.com/dotnet/aspnetcore/pull/41373/files#diff-245d4794033d647784ff2ebfee1d60dfe0b080a2550f37c35282e14cffbb9639)

A newly added trimmable project requires it.